src/components/SimpleSelect/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { SelectHTMLAttributes } from "react";
+import { FieldError, Path, RegisterOptions, UseFormRegister } from "react-hook-form";
+import clsx from "clsx";
+
+import styles from './styles.module.scss';
+
+function SimpleSelect<IFormValues>({ 
+  name, 
+  title, 
+  options,
+  register,
+  validations,
+  className = "",
+  placeholder = "",
+  error
+}: SimpleSelectProps<IFormValues>) {
+  return (
+    <div className={clsx(styles.simpleSelectContainer, "column", className)}>
+      <label 
+        htmlFor={name} 
+        className={clsx(styles.label, "text-normal center m-bottom-1")}
+      >
+        {title}
+      </label>
+      <select id={name} {...register(name, validations)} className={styles.input} defaultValue="">
+        <option value="" disabled hidden>{placeholder}</option>
+        {
+          options.map(({ value, label }) => (
+            <option key={value} value={value}>{label}</option>
+          ))
+        }
+      </select>
+      <p className="text-error italic">{error?.message}</p>
```

Comment:
hey @diego317  you sure you want to display this element even if is undefined? WDTY about

```suggestion
      {!!error && <p className="text-error italic">{error?.message}</p>}
```

Reply:
My idea is that this element should always be there to avoid the growth of the component when an error occurs, as shown in the gif.

Reply:
got it !